lib/preview-web/src/PreviewWeb.tsx

```diff
@@ -127,6 +127,7 @@ export class PreviewWeb<TFramework extends AnyFramework> {
     this.channel.on(Events.RESET_STORY_ARGS, this.onResetArgs.bind(this));
     this.channel.on(Events.FORCE_RE_RENDER, this.onForceReRender.bind(this));
     this.channel.on(Events.FORCE_REMOUNT, this.onForceRemount.bind(this));
+    this.channel.on(Events.STORY_PRELOAD, this.onPreloadStory.bind(this));
```

Comment:
I think the naming for these imperative events would imply this should be `PRELOAD_STORIES`

Reply:
done!